src/py/aspen/config/config.py

```diff
@@ -42,26 +44,38 @@ def AUTH0_CONFIG(self):
 
 
 class Auth0Config:
-    def __init__(self):
-        ENV_FILE = find_dotenv()
-        if ENV_FILE:
-            load_dotenv(ENV_FILE)
+    @property
+    def AWS_SECRET(self):
+
+        secret_name = os.environ.get("SECRET_NAME")
+        region_name = os.environ.get("AWS_REGION")
+
+        session = boto3.session.Session()
+        client = session.client(service_name="secretsmanager", region_name=region_name)
+
+        try:
+            get_secret_value_response = client.get_secret_value(SecretId=secret_name)
+        except ClientError as e:
+            raise e
+        else:
+            secret = get_secret_value_response["SecretString"]
+            return json.loads(secret)
```

Comment:
My guess is that you'll want to cache this value, otherwise you'll keep hitting `secretsmanager`

Reply:
The other thing to be aware of is that these values are slurped into the flask app at startup, and we can't really change them.

Reply:
good points... I'll have to think about how to address the second point

Reply:
@ttung what do you think we should do about the second point? I think this stuff will stay fairly static or the most part, but do you think there's some stuff we should move out of the config?

Reply:
For the second point, I think one possible strategy is just to require that we restart the web services whenever a secret changes.

Reply:
A better strategy might be to fetch the secrets once per web request, and only cache it for the duration of the request.

Reply:
hmmm i'm a little confused, so you think we should decouple auth0 from the config? otherwise these values will be stored in the flask app indefinitely  like what you were saying before

Reply:
i added `lru_cache` decorator to AWS_SECRET which will prevent us from hitting the secret store repeatedly